cypress/integration/flow/Flow.spec.ts

```diff
@@ -149,10 +159,6 @@ describe("Flow", () => {
     });
   });
 
-  it("should create new Flow in hindi", () => {
-    cy.deleteFlow(flow);
-  });
-
```

Comment:
@mdshamoon can undo this change and rename it to delete flow. This is to delete the flow create in previous test. As of now it is creating a flow but not deleting it.